tensortrade/exchanges/exchange.py

```diff
@@ -78,17 +84,30 @@ def is_pair_tradable(self, trading_pair: 'TradingPair') -> bool:
         Returns:
             A bool designating whether or not the pair is tradable.
         """
-        raise NotImplementedError()
+        return str(trading_pair) in self._prices.keys()
 
-    @abstractmethod
     def execute_order(self, order: 'Order', portfolio: 'Portfolio'):
         """Execute an order on the exchange.
 
         Arguments:
             order: The order to execute.
+            portfolio: The portfolio to use.
         """
-        raise NotImplementedError
+        trade = self._service(
```

Comment:
This line is a bit confusing. It seems like it should be `self._service.execute_order(...)`